tests/nb/light_HassTurnOn.yaml

```diff
@@ -1,6 +1,69 @@
 language: 'nb'
 tests:
-- sentences: []
-  intent:
-    name: HassTurnOn
-    slots: {}
+  - sentences:
```

Comment:
Already merged in in #904. Not required.

Reply:
Thanks. But still a lot more to test, so will keep my changes.